azure-mgmt-batch/src/main/java/com/microsoft/azure/management/batch/Application.java

```diff
@@ -0,0 +1,190 @@
+/**
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License. See License.txt in the project root for
+ * license information.
+ */
+
+package com.microsoft.azure.management.batch;
+
+import com.microsoft.azure.management.apigeneration.LangDefinition;
+import com.microsoft.azure.management.batch.implementation.ApplicationInner;
+import com.microsoft.azure.management.batch.implementation.ApplicationPackageInner;
+import com.microsoft.azure.management.resources.fluentcore.arm.models.ExternalChildResource;
+import com.microsoft.azure.management.resources.fluentcore.model.Attachable;
+import com.microsoft.azure.management.resources.fluentcore.model.Settable;
+import com.microsoft.azure.management.resources.fluentcore.model.Wrapper;
+
+import java.util.List;
+
+/**
+ * An immutable client-side representation of an Azure batch account application.
+ */
+@LangDefinition(ContainerName = "~/")
+public interface Application extends
+        ExternalChildResource<Application>,
+        Wrapper<ApplicationInner> {
+
+    /**
+     * Get the displayName value.
+     *
+     * @return the displayName value
+     */
+    String displayName();
+
+    /**
+     * @return the list of application packages
+     */
+    List<ApplicationPackageInner> packages();
+
+    /**
+     * Get the allowUpdates value.
+     *
+     * @return the allowUpdates value
+     */
+    boolean allowUpdates();
```

Comment:
i suspect this is a property , not a method (i.e. it doesn't actually do anything it just returns a value). If that's the case, then this should not be using a verb in its name making it sounds like a method. Instead, something like: `updatesAllowed()`

Also, the Javadoc should be more descriptive, like: "true if updates are allowed, otherwise false"